src/components/ChatEntry.js

```diff
@@ -1,38 +1,35 @@
 import React from 'react';
-import { useState } from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
 import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
-  const [color, setColor] = useState(false);
-
-  const handleClick = () => {
-    if (color === true) {
-      setColor(false)
-      props.decrementLikes()
-    } else if (color === false) {
-      setColor(true)
-      props.incrementLikes()
-    }
+  const handleClick = (event) => {
+    props.onLikeClicked(props.id)
   }
- 
+
+  const chatName = (props.sender === 'Vladimir') ? 'local':'remote';
   return (
-    <div className="chat-entry local">
+  <div>
+    <div className={`chat-entry ${chatName}`}>
       <h2 className="entry-name">{props.sender}</h2>
       <section className="entry-bubble">
         <p>{props.body}</p>
         <p className="entry-time"><TimeStamp time={props.timeStamp}/></p>
-        <button className="like" onClick={handleClick}>{!color ? '🤍' : '❤️'}</button>
+        <button className="like" onClick={handleClick}>{props.liked ? '❤️' : '🤍'}</button>
```

Comment:
👍 Now the liked display is driven by the props (being passed in) rather than internally tracked state. The component can focus on displaying the data it is given, rather than needing to manage that data itself.